cirrus/server/cirrus/main.py

```diff
@@ -109,6 +110,39 @@ def initialize_glean():
     return pings, metrics
 
 
+def get_metrics_data(
```

Comment:
`get_metrics` over `get_metrics_data` imo.

Also this seems to be returning information about enrollments, not metrics?

Reply:
It's collating the data that's fed into enrollment metrics, I'm going to rename it `collate_enrollment_metric_data` I think